### PR TITLE
Feature/young's modulus

### PR DESCRIPTION
**Pull request by MaartenBehn**

### CODE REVIEW

File: distance3d/hydroelastic_contact/_tetrahedron_intersection.py [hunk 1]
```diff
@@ -230,10 +250,10 @@ def check_tetrahedra_intersect_contact_plane(
     plane_distances1 = tetrahedron1.dot(plane_normal) - d
     plane_distances2 = tetrahedron2.dot(plane_normal) - d
     return (
-        min(plane_distances1) < -tolerance
-        and max(plane_distances1) > tolerance
-        and min(plane_distances2) < -tolerance
-        and max(plane_distances2) > tolerance)
+            min(plane_distances1) < -tolerance
```
Comment (AlexanderFabisch) [inline distance3d/hydroelastic_contact/_tetrahedron_intersection.py:253]: Why do we have to change this?

Comment (MaartenBehn) [reply distance3d/hydroelastic_contact/_tetrahedron_intersection.py:253]: Oh, that was just auto formatting.

File: distance3d/hydroelastic_contact/_rigid_body.py [hunk 2]
```diff
@@ -37,6 +37,8 @@ def __init__(self, body2origin, vertices, tetrahedra, potentials):
         self._aabbs = None
         self._aabb_tree = None
 
+        self._youngs_modulus = 1.0
```
Comment (AlexanderFabisch) [inline distance3d/hydroelastic_contact/_rigid_body.py:40]: I'd suggest to add it as a constructor argument with default value 1.0

**Comment by MaartenBehn** [comment]
Is ready to be reviewed for merge.

File: examples/visualizations/vis_pressure_field_collision_multible_objects.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
+"""
+=======================================================================
+Physical simulation exploring the young's modulus with multiple Objects
```
Comment (AlexanderFabisch) [inline examples/visualizations/vis_pressure_field_collision_multible_objects.py:3]: capitalization is a bit strange here

File: examples/visualizations/vis_youngs_modulus.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
+"""
+=================================================
+Visualize Pressure Field of Two Colliding Objects
```
Comment (AlexanderFabisch) [inline examples/visualizations/vis_youngs_modulus.py:3]: Title should reflect what is happening in the script

File: distance3d/test/test_hydroelastic_contact.py [hunk 5]
```diff
@@ -251,3 +251,43 @@ def test_tetrahedral_mesh_volumes():
...
+
+    intersection_3, wrench12_3, _, _ = hydroelastic_contact.contact_forces(
+        rigid_body1, rigid_body2, return_details=True)
+
+    assert intersection_3
+
+    expected_vals = [0, 0.00032, 0, 0, 0, 0]
+
+    assert ((wrench12_3 - expected_vals) < 0.00001).all()
+    len_2 = np.sqrt(wrench12_3[0] * wrench12_3[0]
```
Comment (AlexanderFabisch) [inline distance3d/test/test_hydroelastic_contact.py:286]: You can use np.linalg.norm

File: distance3d/test/test_hydroelastic_contact.py [hunk 6]
```diff
@@ -251,3 +251,43 @@ def test_tetrahedral_mesh_volumes():
...
+        rigid_body1, rigid_body2, return_details=True)
+
+    rigid_body1.youngs_modulus = 1
+    rigid_body2.youngs_modulus = 1
+
+    intersection_2, wrench12_2, wrench21_2, _ = hydroelastic_contact.contact_forces(
+        rigid_body1, rigid_body2, return_details=True)
+
+    assert intersection1 == intersection_2
+    assert (wrench12_1 == wrench12_2).all()
```
Comment (AlexanderFabisch) [inline distance3d/test/test_hydroelastic_contact.py:272]: You can use numpy.testing.assert_array_(almost_)equal

**Pull request merged**